test/tests/postprocessors/constant/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [receiver]
+    type = CSVDiff
+    input = receiver.i
+    csvdiff = receiver_out.csv
+    requirement = "The system shall allow the value of a ConstantPostprocessor to be controllable"
```

Comment:
```suggestion
    requirement = "The system shall allow the value of a postprocessor to be controllable."
```
Requirements should be agnostic of implementation details. In theory, the requirement should come first, and then we implement the capability that accomplishes the requirement